statistics/builder.go

```diff
@@ -29,15 +29,17 @@ type SortedBuilder struct {
 	bucketIdx       int64
 	Count           int64
 	hist            *Histogram
+	statsVer        int
```

Comment:
`statsVer` seems confused, how about renaming it to `recordBucketNDV` or `useBucketNDV`?

Reply:
It will be used to specify the future analyze versions too?
You mean, when we introduce new versions, we use new field to control the new behavior?